nbconvert/templates/latex/base.tplx

```diff
@@ -161,7 +163,7 @@ This template does not define a docclass, the inheriting class must define this.
 
     ((* block commands *))
     % Prevent overflowing lines due to hard-to-break entities
-    \sloppy 
+    \sloppy
```

Comment:
I would see if you can turn off your auto-formatter for .tplx and .tpl files. There are situations in which having spaces can be semantically meaningful in LaTeX.